### PR TITLE
Refactor and lint files

### PR DESCRIPTION
**Pull request by chubberlisk**

### CODE REVIEW

File: CryptoTechReminderSystem.Test/UseCase/RemindLateDevelopersTests.cs [hunk 1]
```diff
@@ -10,9 +10,9 @@ namespace CryptoTechReminderSystem.Test.UseCase
 {
     public class RemindLateDevelopersTests
     {
-        SendReminderSpy _sendReminderSpy;
-        GetLateDevelopersSpy _getLateDevelopersSpy;
-        GetLateDevelopersStub _getLateDevelopersStub;
+        private SendReminderSpy _sendReminderSpy;
+        private GetLateDevelopersSpy _getLateDevelopersSpy;
+        private GetLateDevelopersStub _getLateDevelopersStub;
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Test/UseCase/RemindLateDevelopersTests.cs]: arent these private by default?

File: CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs [hunk 2]
```diff
@@ -40,5 +32,15 @@ public void Execute(RemindLateDevelopersRequest remindLateDevelopersRequest)
                 }
             }
         }
+        
+        private bool IsHalfHourInterval()
+        {
+            return _clock.Now().ToUnixTimeSeconds() % 1800 == 0;
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs]:
```suggestion
            return _clock.Now().Minute % 30 == 0;
```

File: CryptoTechReminderSystem.Test/UseCase/RemindLateDevelopersTests.cs (hunk 1 above)
Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Test/UseCase/RemindLateDevelopersTests.cs]: Yes.

Source: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/accessibility-levels

File: CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs (hunk 2 above)
Comment (dushan-madetech) [reply CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs]: I disagree good sir!

Comment (RobertM905) [reply CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs]: 🥇

Comment (yusufsheiqh) [reply CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs]: Resolved in https://github.com/chubberlisk/crypto_tech/pull/35/commits/001e1c2bf7a516378e65ab0e5a29b143b5e2fec6.

File: CryptoTechReminderSystem.Test/UseCase/RemindDeveloperTests.cs [hunk 3]
```diff
@@ -38,7 +39,7 @@ public void CanRemindDeveloper2()
         {
             var spy = new SlackGatewaySpy();
             var remindDeveloper = new SendReminder(spy);
-            var text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
+            const string text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
```
Comment (yusufsheiqh) [inline CryptoTechReminderSystem.Test/UseCase/RemindDeveloperTests.cs]: Extracted into a `private const string` as it is used multiple times in this test suite.

Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Test/UseCase/RemindDeveloperTests.cs]: Solved in https://github.com/chubberlisk/crypto_tech/pull/35/commits/a5de69c15e5871e50fa7be6312071e9d119792f7.

**Pull request merged**